apps/omg_watcher/test/db/txoutput_db_test.exs

```diff
@@ -0,0 +1,241 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.DB.TxOutputDBTest do
```

Comment:
shouldn't you test more on the public API here? the public APIs could be the stuff that block getter calls to put stuff into the DB and then the API endpoints behaving correctly?

Reply:
:question: